search/Property.cc

```diff
@@ -889,6 +889,7 @@ getProperty(const Instance *inst,
 	    Sta *sta)
 {
   auto network = sta->cmdNetwork();
+  LibertyCell *libertyCell = network->libertyCell(inst);
```

Comment:
variables should use snake case
see doc/CodingGuidelines.txt

Reply:
Sorry will fix